searx/webapp.py

```diff
@@ -114,16 +111,22 @@
 )
 from searx.flaskfix import patch_application
 
-# renaming names from searx imports ...
+from searx.locales import (
+    LOCALE_NAMES,
+    RTL_LOCALES,
+    localeselector,
+    initialize_locales,
+)
 
+# renaming names from searx imports ...
 from searx.autocomplete import search_autocomplete, backends as autocomplete_backends
 from searx.languages import language_codes as languages
-from searx.locales import LOCALE_NAMES, RTL_LOCALES
 from searx.search import SearchWithPlugins, initialize as search_initialize
 from searx.network import stream as http_stream, set_context_network_name
 from searx.search.checker import get_result as checker_get_result
 
 logger = logger.getChild('webapp')
+initialize_locales()
```

Comment:
I would move this line just above https://github.com/searxng/searxng/blob/59ef9b9287f1beda12f7b9a20b93cbc378a22bac/searx/webapp.py#L1418-L1419

so all the initialization calls are grouped in one place.

Reply:
> I would move this line just between lines 1417 and 1418

Done .. but now we have [this issue](https://github.com/searxng/searxng/runs/6848643803?check_suite_focus=true#step:7:163) in the robot test .. TBH the inititialization of the app is not clear to me / do you have an idea what the problem is in robot-test?

> At least wikidata seems to return traditional Chinese when the search language is "中文 (中國香港)", but I'm not Chinese reader.

FYI: #1303 is about UI language .. and this PR adds a dictionary to map *best match*:

> In the searx.locales module a new dictionary named `LOCALE_BEST_MATCH` has been added. In this dictionary we can map languages without a translation to languages we have a translation of.